lib/tasks/rest_dial.js

```diff
@@ -28,6 +28,7 @@ class TaskRestDial extends Task {
   */
   async exec(cs) {
     await super.exec(cs);
+    this.req = cs.req;
```

Comment:
where is `this.req` actually referenced?

Reply:
In the kill method. It attempts to use the passed in call session, if that isn't available the saved request is used.